rewrite-properties/src/main/java/org/openrewrite/properties/tree/Properties.java

```diff
@@ -55,7 +55,9 @@ default <P> boolean isAcceptable(TreeVisitor<?, P> v, P p) {
 
     @lombok.Value
     @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
-    class File implements Properties, SourceFile {
+    @RequiredArgsConstructor
+    @AllArgsConstructor
```

Comment:
Xml.Document uses 
```java
    @AllArgsConstructor(access = AccessLevel.PRIVATE)
```
What's the reason behind not copying that approach here?

Reply:
Fixed